bazel/protobuf.patch

```diff
@@ -11,6 +11,113 @@ index 1690d4219..8a7f1bf14 100644
  
  ################################################################################
  # Protobuf Runtime Library
+@@ -510,14 +510,105 @@ cc_library(
+     deps = [":protobuf"],
+ )
+ 
++
+ cc_binary(
+-    name = "protoc",
++    name = "compiled_protoc",
+     srcs = ["src/google/protobuf/compiler/main.cc"],
+     linkopts = LINK_OPTS,
+     visibility = ["//visibility:public"],
+     deps = [":protoc_lib"],
+ )
+ 
++
++# Lifted from `rules_proto`
```

Comment:
Any reason not use rules_proto directly?

Reply:
we dont install their deps, and in the many attempts at trying one way or another it didnt work

also, our current rules_proto's protobuf version is incompatible with ours, so this seemed the easiest way all round